hawkular-alerts-bus/src/main/java/org/hawkular/alerts/bus/init/CacheManager.java

```diff
@@ -46,98 +51,236 @@
 public class CacheManager {
     private final Logger log = Logger.getLogger(CacheManager.class);
 
-    //public static final String CACHE_KEY_TRIGGER_UPDATE_TIME = "HawkularAlerts:TriggerUpdateTime";
-    //public static final String CACHE_KEY_CONDITION_UPDATE_TIME = "HawkularAlerts:ConditionUpdateTime";
-    //public static final String CACHE_KEY_DAMPENING_UPDATE_TIME = "HawkularAlerts:DampeningUpdateTime";
+    // private final String ALERTING = "alerting";
 
-    Set<String> activeDataIds;
-    //long activeDataIdsTime = 0L;
-    Set<String> activeAvailabityIds;
+    private final String AVAILABILITY = "availability";
+    private final String GAUGE = "gauge";
+    private final String COUNTER = "counter";
+    private final String COUNTER_RATE = "counter_rate";
+    private final String STRING = "string";
+    private final String GAUGE_RATE = "gauge_rate";
```

Comment:
As discussed in the other PR, I think these constants should be more unique to avoid conflict with non-hmetrics dataids.